app/src/main/java/ca/wheresthebus/ui/home/AddFavBottomSheet.kt

```diff
@@ -73,10 +85,10 @@ class AddFavBottomSheet : BottomSheetDialogFragment() {
                     // Update the search bar text and perform search every time the text changes
                     val searchText = charSequence?.toString() ?: ""
                     if (searchText.isBlank()) {
-                        nearbySuggestionsRecyclerView.visibility = View.VISIBLE
+                        nearbySuggestionsView.visibility = View.VISIBLE
                         stopSuggestionsView.visibility = View.INVISIBLE
                     } else {
-                        nearbySuggestionsRecyclerView.visibility = View.INVISIBLE
+                        nearbySuggestionsView.visibility = View.INVISIBLE
```

Comment:
INVISIBLE takes up space in the layout, but just doesn't render. Might want to prefer View.GONE instead, but this is just a nitpick.
See: https://stackoverflow.com/questions/11556607/android-difference-between-view-gone-and-view-invisible